agenta-cli/agenta/cli/variant_commands.py

```diff
@@ -299,7 +299,7 @@ def remove_variant(variant_name: str, app_folder: str, host: str):
     config_file = Path(app_folder) / "config.toml"
     config = toml.load(config_file)
     app_name = config["app_name"]
-    api_key = config.get("api_key", None)
+    api_key = config.get("api_key", "")
```

Comment:
we have a lot of checks like :
headers={"Authorization": api_key} if api_key is not None else None,

I thinks will make some of checks not valid. 

@devgenix can you please check?

Reply:
I checked and this was the last one having None.